.github/workflows/staging.yml

```diff
@@ -1,9 +1,7 @@
 name: Staging
 on:
   push:
-    branches: [develop]
-  pull_request:
-    branches: [develop]
```

Comment:
Please keep this pull request to `[main]` action

Reply:
@wi1dcard If we keep this pull request to main, will it do deploy when new pull request are created?

Or it will do when new pull request are closed?

Reply:
@JayJay1024 As you said on slack, ee do not deploy on pull request.

But we still need to have an ci for pull request